### ISSUE TITLE
Nix on WSL killing processes unnecessarily

### ISSUE THREAD

**Issue by milibopp** [NixOS/nix#2714]
Using Nix on the Windows Subsystem for Linux (WSL) when building Nix packages or switching my home-manager configuration, I always see a few messages like this

    killing process 9146: No such process

This is not a functional issue, but I was wondering whether something is wrong, as I have never seen this on a regular NixOS installation.
For reference, I believe #2292 shows the same issue, although this is not the focus of that issue.

**Comment by Gerschtli** [comment]
Are there any updates? I'm interested in this issue too.

**Comment by aepsil0n** [comment]
@Gerschtli this is actually not happening any more for me. I forgot I had even opened this issue. Are you seeing this, too? Do you think it should remain open?

**Comment by Gerschtli** [comment]
Yes, I am still seeing these messages with latest nix installation.

**Comment by 573** [comment]
> 
> 
> Yes, I am still seeing these messages with latest nix installation.

Can confirm for my WSL nix installation as well.

**Comment by szabi** [comment]
Can confirm as of today, current nix installer and Windows 10 Enterprise build 17134.

**Comment by tomberek** [comment]
Closing as stale. Re-open if needed.
